### PR TITLE
test: Fedora error/fail logs should not be a must

### PR DESCRIPTION
**Pull request by yuxisun1217**
pcieport 0000:00:01.6: pciehp: Failed to check link status
RAS: Correctable Errors collector initialized
These 2 logs should not be must existing logs in dmesg.

See: https://gitlab.com/osbuild/ci/osbuild-composer/-/jobs/1429591834#L3392

Signed-off-by: Yuxin Sun <yuxisun@redhat.com>

This pull request includes:

- [x] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] create a file in [news/unreleased](https://github.com/osbuild/osbuild-composer/tree/main/docs/news/unreleased) directory if this change should be mentioned in the release news
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

File: test/data/ansible/check_ostree.yaml
Comment (atodorov) [inline test/data/ansible/check_ostree.yaml:332]: This was raised recently on another PR, IIRC by @ondrejbudai. 

At this point do we even care to check dmesg output ? It starts to feel a bit fragile. 

need_info @henrywang, @yih-redhat

Comment (atodorov) [reply test/data/ansible/check_ostree.yaml:332]: @henrywang why make #1564 as a separate PR instead of reusing the current PR and/or have @yuxisun1217 just delete the relevant lines ?

Comment (atodorov) [reply test/data/ansible/check_ostree.yaml:332]: FTR I also don't see a public discussion about the previous question.

Comment (yuxisun1217) [reply test/data/ansible/check_ostree.yaml]: I just deleted all the dmesg log checking tasks follows #1564 . Let's wait for @ondrejbudai 's reply.

Comment (atodorov) [reply test/data/ansible/check_ostree.yaml]: I don't follow here. Why not introduce the deletion as a separate commit in this pull request instead of opening new PR and having to wait for it before continuing work on this one ?

Comment (ondrejbudai) [reply test/data/ansible/check_ostree.yaml]: > I just deleted all the dmesg log checking tasks follows #1564 . Let's wait for @ondrejbudai 's reply.

I'm sorry but I'm not sure what kind of reply is expected from me. :D

Comment (yuxisun1217) [reply test/data/ansible/check_ostree.yaml]: Hmmm I think we still can see the previous change in this conversation, although it is marked as outdated. It will be a little complex to merge if I create multiple commit in one PR and many maintainers don't like that I think.

**Comment by yuxisun1217** [comment]
The CI jobs are failed because the PSI openstack issue:
```
Error: Error waiting for instance (0c31092b-9244-4b9f-b924-b8a2e3172fef) to become ready: unexpected state 'ERROR', wanted target 'ACTIVE'. last error: %!s(<nil>)
```
Need to wait for a while.

File: test/data/ansible/check_ostree.yaml
Comment (atodorov) [reply test/data/ansible/check_ostree.yaml]: > It will be a little complex to merge if I create multiple commit in one PR and many maintainers don't like that I think.

This is quite the opposite, we rarely have pull requests containing a single commit only. Most of them have quite a few commits. The governing principle is that those commits are logically related to achieve the functionality for which the PR was opened.

Comment (yuxisun1217) [reply test/data/ansible/check_ostree.yaml]: Understand. I'll be aware of that the next time. Thanks @atodorov !

Comment (yuxisun1217) [reply test/data/ansible/check_ostree.yaml]: > > I just deleted all the dmesg log checking tasks follows #1564 . Let's wait for @ondrejbudai 's reply.
> 
> I'm sorry but I'm not sure what kind of reply is expected from me. :D

Hi @ondrejbudai ,
We may need you to confirm "At this point do we even care to check dmesg output ? It starts to feel a bit fragile." (see the first comment by Alex, if this checkpoint is added by you :D )

Comment (ondrejbudai) [reply test/data/ansible/check_ostree.yaml]: Ah, I can't remember, sorry. I am all for removing these checks, testing log output can be tricky.

Comment (yuxisun1217) [reply test/data/ansible/check_ostree.yaml]: OK thanks @ondrejbudai ! 

Hi @atodorov , could you please help to merge this PR? Thanks!

**Review by henrywang — APPROVED** [review]
LGTM. Thanks @yuxisun1217.

**Pull request merged**
